### PR TITLE
Dom todos

### PR DESCRIPTION
**Pull request by jjshab**
All set for review

### CODE REVIEW

File: app/src/main/java/com/gdgbaltimore/bmoreart/HomeActivity.java [hunk 1]
```diff
@@ -15,18 +23,32 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.activity_home);
 
         SupportMapFragment mapFragment = (SupportMapFragment) getSupportFragmentManager()
-                .findFragmentById(R.id.map);
```
Comment (zerobasedindex) [inline app/src/main/java/com/gdgbaltimore/bmoreart/HomeActivity.java]: Usually you don't static import layout resource ids. My guess the IDE did this automagically, but please change it back to `R.id.map`

File: app/src/main/java/com/gdgbaltimore/bmoreart/HomeActivity.java [hunk 2]
```diff
@@ -15,18 +23,32 @@ protected void onCreate(Bundle savedInstanceState) {
...
+        googleMap.addMarker(new MarkerOptions().position(new LatLng(39.2818289, -76.584183)));
+        googleMap.addMarker(new MarkerOptions().position(new LatLng(39.408765, -76.596545)));
+        googleMap.addMarker(new MarkerOptions().position(new LatLng(39.3266504, -76.6048989)));
 
-        // Add at least 3 pins, maybe OrderUp HQ, Goucher College, and pick a restaurant
 
         // Draw a polyline around the Baltimore city boundry
         // You'll have to google to get lat/lon points for that boundry
+
+        Polyline line = googleMap.addPolyline(new PolylineOptions().add(new LatLng(39.372264, -76.711529), new LatLng(39.372264, -76.529649), new LatLng(39.207672, -76.532966),
```
Comment (zerobasedindex) [inline app/src/main/java/com/gdgbaltimore/bmoreart/HomeActivity.java]: Can you think of a better way to store these lat/lngs? It's typically bad form to inline/hard code data like this into the middle of your code. You should look at pulling this out into a member variable on the activity. (google Java class member variables).

You could use an array list to store them: http://www.anyexample.com/programming/java/java_arraylist_example.xml

Or an array: https://www.tutorialspoint.com/java/java_arrays.htm

```
private static final LatLng[] BALT_REGION = new LatLng[]{new LatLng(39.207579, -76.583704), ...};
```

In the end, the code is just being moved but it's considered to be "in a better place"

**Review by zerobasedindex — CHANGES_REQUESTED** [review]
Looking good man, two minor comments. Push a fix and then we can merge!

File: app/src/main/java/com/gdgbaltimore/bmoreart/HomeActivity.java (hunk 1 above)
Comment (jjshab) [reply app/src/main/java/com/gdgbaltimore/bmoreart/HomeActivity.java]: got it

File: app/src/main/java/com/gdgbaltimore/bmoreart/HomeActivity.java (hunk 2 above)
Comment (jjshab) [reply app/src/main/java/com/gdgbaltimore/bmoreart/HomeActivity.java]: I could only come up with this, but it doesn't work. I have no idea how java variables work at all.
ArrayList<LatLng> coordlist = new ArrayList<LatLng>();

        coordlist.add(new LatLng(39.372264, -76.711529));
        coordlist.add(new LatLng(39.372264, -76.529649));
        coordlist.add(new LatLng(39.207672, -76.532966));
        coordlist.add(new LatLng(39.197325, -76.550081));
        coordlist.add(new LatLng(39.207578, -76.583704));
        coordlist.add(new LatLng(39.234661, -76.611865));
        coordlist.add(new LatLng(39.277896, -76.711035));
        coordlist.add(new LatLng(39.372264, -76.711529));



        Polyline line = map.addPolyline(new PolylineOptions()
                .add(coordlist)
                .width(5)
                .color(Color.RED));

Comment (zerobasedindex) [reply app/src/main/java/com/gdgbaltimore/bmoreart/HomeActivity.java]: You should be able to do `.add(coordlist.toArray(new LatLng[]{}))`

**Comment by zerobasedindex** [comment]
@jjshab I went ahead a pushed a commit for you, please review it and let me know if you have any questions. Figured this was a good place for an example to learn from.

**Review by zerobasedindex — APPROVED** [review]

**Pull request merged**